src/server/svr_func.c

```diff
@@ -3265,8 +3263,7 @@ set_entity_ct_sum_queued(job *pjob, pbs_queue *pque, enum batch_op op)
 	if ((pjob->ji_qs.ji_state == JOB_STATE_MOVED) ||
 		(pjob->ji_qs.ji_state == JOB_STATE_FINISHED) ||
 		(pjob->ji_qs.ji_state == JOB_STATE_EXPIRED) ||
-		((pjob->ji_qs.ji_state == JOB_STATE_RUNNING) && (op == INCR)) ||
-		((pjob->ji_entity_limit_set == 0) && (op == DECR)))
```

Comment:
removed `((pjob->ji_entity_limit_set == 0) && (op == DECR))` as it created phantom usage when multiple queued limits are in force.